wrappers/java/src/test/java/org/hyperledger/indy/sdk/crypto/PackUnpackMessageTest.java

```diff
@@ -77,14 +78,20 @@ public void testPackMessageErrorsInvalidReceivers() throws Exception {
 	}
 
 	@Test
-	public void testUnpackMessageErrorsWithInvalidPackedMessageStructure() throws Exception {
+	public void testUnpackMessageErrorsWithInvalidPackedMessageStructure() {
 		String packedMessage = "jibberish";
 
 		thrown.expect(ExecutionException.class);
 		thrown.expectCause(isA(InvalidStructureException.class));
 
-		Crypto.unpackMessage(wallet, packedMessage.getBytes()).get();
-
+		try {
```

Comment:
Should this test just expect InvalidParameterException instead of invalid structure few lines above and keep old test body?